lxd/main_cluster.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	lxd "github.com/lxc/lxd/client"
+	"github.com/lxc/lxd/lxd/cluster"
+	"github.com/lxc/lxd/lxd/db"
+	"github.com/lxc/lxd/lxd/sys"
+	"github.com/lxc/lxd/shared"
+	"github.com/lxc/lxd/shared/i18n"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type cmdCluster struct {
+	global *cmdGlobal
+}
+
+func (c *cmdCluster) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = "cluster"
+	cmd.Short = "Low-level cluster administration commands"
+	cmd.Long = `Description:
+  Low level administration tools for inspecting and recovering LXD clusters.
+`
+	// List database nodes
+	listDatabaseNodes := cmdClusterListDatabaseNodes{global: c.global}
+	cmd.AddCommand(listDatabaseNodes.Command())
+
+	// Recover
+	recover := cmdClusterRecoverFromQuorumLoss{global: c.global}
+	cmd.AddCommand(recover.Command())
+
+	return cmd
+}
+
+type cmdClusterListDatabaseNodes struct {
+	global *cmdGlobal
+}
+
+func (c *cmdClusterListDatabaseNodes) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = i18n.G("list-database-nodes")
```

Comment:
Any reason not to call this just `list`?

Also, we don't say `nodes` anywhere user visible, usually using `cluster members` instead.

Reply:
Yeah so ```list-database-nodes``` is to make it clear that is this is a list of only the cluster members that are acting as database nodes, not all members.

I'm using ```nodes``` to highlight the difference: a cluster member might a be a database node or not. Calling the command ```list-database-members``` feels odd, and calling it just ```list``` feels potentially confusing.